console/account/how-to/use-the-search-bar.mdx

```diff
@@ -0,0 +1,37 @@
+---
+meta:
+  title: How to use the search bar
+  description: This page explains how to use the search bar
+content:
+  h1: How to use the search bar
+  paragraph: This page explains how to use the search bar
+tags: account search-bar resource
+dates:
+  validation: 2024-03-14
+  posted: 2024-03-14
+categories:
+  - account
+---
+
+The **search bar** functionality within the Scaleway console allows you to quickly find resources.
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com).
+</Message>
+
+1. Log into the [Scaleway console](https://console.scaleway.com).
+2. Navigate to the search bar located next to the organization dashboard menu.
```

Comment:
```suggestion
2. Navigate to the search bar located next to the Organization dashboard menu.
```

Reply:
Is this feature available in the console? I've tried to find the search bar as instructed in the steps above but I can't find it

Reply:
> Is this feature available in the console? I've tried to find the search bar as instructed in the steps above but I can't find it

not yet.. but it will be soon :)))